client/components/domains/domain-registration-suggestion/index.jsx

```diff
@@ -28,32 +28,42 @@ const DomainRegistrationSuggestion = React.createClass( {
 
 	render() {
 		const { suggestion, translate } = this.props,
-			isAdded = hasDomainInCart( this.props.cart, suggestion.domain_name ),
+			domain = suggestion.domain_name,
+			isAdded = hasDomainInCart( this.props.cart, domain ),
 			domainFlags = [];
+
 		let buttonClasses, buttonContent;
 
-		if ( suggestion.domain_name ) {
+		if ( domain ) {
 			const newTLDs = [];
+			const a8cTLDs = [ '.ca', '.de', '.fr' ];
+			const tld = domain.substring( domain.indexOf( '.' ) );
```

Comment:
The `endsWith` was there for TLDs like `co.uk`. But I think this one is ok until we add those.

If we're using `indexOf(.)` we can probably just have the TLDs in the array w/o it.